apps/music/elements/music-search-box.js

```diff
@@ -200,14 +159,4 @@ function debounce(fn, ms) {
   };
 }
 
-try {
```

Comment:
It was for catching a custom-element being registered twice in the same document.